app/src/types/components.interface.ts

```diff
@@ -187,8 +187,20 @@ export interface HubConfig {
   trade_limit: number
 }
 
-export interface Denom {
-  native: string
+export type Addr = string
```

Comment:
Do we have any benefit in having an `Addr` type since it is the same as string? Should we replace all the places where we use address and the type is "string" with "Addr"?  

I like the idea of having an `Addr` type, I am just thinking of the usage and needs